webpack.config.js

```diff
@@ -39,7 +43,7 @@ module.exports = {
             },
             {
                 test: /\.css$/,
-                loader: "style!css"
+                loader: ExtractTextPlugin.extract('style-loader', 'css-loader')

```

Comment:
Since we're changing the way we work with css, let's add postcss/sass support. Here are some snippets:
`{
                        test: /\.css$/,
                        loader: ExtractTextPlugin.extract('style-loader', [
                            'css-loader',
                            'postcss-loader'
                        ].join('!'))
                    }`
`{
                        test: /\.scss$/,
                        loader: ExtractTextPlugin.extract('style-loader', [
                            'css-loader',
                            'postcss-loader',
                            ``sass-loader?includePaths[]=${pathResolver.client()}``
                        ].join('!'))
                    }`
`postcss: [
                autoprefixer({
                    browsers: ['last 2 versions']
                })
            ]`